src/java.com/homework6/Flower.java

```diff
@@ -0,0 +1,10 @@
+package homework6;
+
+public class Flower {
```

Comment:
Please add field price here and constructor with parameter(s). 
Also, what do you think about making this class abstract?